package.json

```diff
@@ -72,6 +72,7 @@
     "raf": "^3.4.0",
     "react": "^16.0.0",
     "react-bootstrap": "^0.31.0",
+    "react-bootstrap-table": "^4.1.5",
```

Comment:
I'm in mixed feelings about this library - didn't  you prefer the other one? :)

Reply:
The other library doesn't use bootstrap table classes, and more then that it doesn't use tables tags at all , so it would be difficult to add bootstrap and patternfly style.